# key for defining type of data and float vector for values
    float32[] Q_cost
    float32[] Qn_cost
    float32[] R_cost
    float32 Ts
    float32 Tf
    float32 N
    float32 s_ref
    float32 v_ref
    bool lifted
    float32 ns
